src/order-history/service.js

```diff
@@ -76,7 +51,7 @@ export async function getOrders(page = 1, pageSize = 20) {
 
     return {
       datePlaced: date_placed, // eslint-disable-line camelcase
-      total: total_excl_tax, // eslint-disable-line camelcase
+      total: isNotDecimalish(total_excl_tax) ? total_excl_tax : `$${total_excl_tax}`, // eslint-disable-line camelcase
```

Comment:
The Legacy Ecomm system works in USD, the modern does Remote Currency formatting. Thus we could get bacl `$12.34 AUD` instead of `12.34`.